server/src/main/scala/Routes.scala

```diff
@@ -0,0 +1,17 @@
+package server
+
+import akka.http.scaladsl.server.{Directives, Route}
+import java.io.File
+
+object Routes extends Directives {
+  def apply(): Route =
+    pathSingleSlash(getFrom("index.html")) ~
+    path(Segment)(getFrom) ~
+    path("target" / Segment)(s => getFrom(s"target/$s"))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal " appears 3 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594876549/issues/source?bid=3301982&fileBranchId=3306761#l10)